bot/src/infrastructure/http/MapClient.ts

```
@@ -1,155 +1,193 @@
+  MapActivityResponseSchema, 
   UserCharactersResponseSchema,
-} from "../../types/ingestion";
-import { logger } from "../../lib/logger";
+  MapActivityResponse,
+  UserCharactersResponse 
+} from '../../shared/schemas/api-responses';
+import { logger } from '../../lib/logger';
+import { RateLimiter } from '../../shared/performance/rateLimiter';
+import { rateLimiterManager } from '../../shared/performance/RateLimiterManager';
+import { ExternalServiceError, ValidationError } from '../../shared/errors';
+import * as crypto from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)